Manito/Manito/Presentation/Scene/LoginScene/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  LoginViewController.swift
+//  Manito
+//
+//  Created by Mingwan Choi on 2022/07/07.
+//
+
+import Combine
+import UIKit
+
+import SnapKit
+
+final class LoginViewController: UIViewController {
+
+    // MARK: - ui component
+
+    private let loginView: LoginView = LoginView()
+    
+    // MARK: - property
+    
+    private var cancellable = Set<AnyCancellable>()
```

Comment:
p1;

type annotation 부탁드립니다. 
```suggestion
    private var cancellable: Set<AnyCancellable> = Set()
```

Reply:
[046fc07](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/046fc07d03e9b0d0e859e1afc3ce8a141c234cd8)
반영하였습니다.